specification.md

```diff
@@ -0,0 +1,1199 @@
+      }
+    }
+  ]
+}
+```
+
+Additionally, there is a data file containing the data described by this resource, in this example, `data/flow-results-example-1-data.json`. This file provides all individual Responses in the following JSON format:
+
+```
+[
+  [ "2017-05-23T13:35:37-04:00", 20394823948, 923842093, "ae54d3", "female", {"option_order": ["male","female"]} ],
+  [ "2017-05-23T13:35:47-04:00", 20394823950, 923842093, "ae54d7", "chocolate", {} ],
+  [ "2017-05-24T15:15:37-04:00", 20394823952, 923842086, "ae54d3", "male", {"option_order": ["male","female"]} ],
+  [ "2017-05-23T15:16:12-04:00", 20394823953, 923842086, "ae54d7", "vanilla", {} ],
+  [ "2017-05-23T15:16:20-04:00", 20394823954, 923842086, "ae54d8", 196, {} ],
+  [ "2017-05-23T15:16:38-04:00", 20394823955, 923842086, "ae54da", "I am feeling curious.", {"type": "text", "language": "eng"} ],
+  [ "2017-05-23T17:25:12-04:00", 20394823956, 923842093, "ae54da", "https://myexampleflowserver.org/resources/audio/20394823956.ogg", {"type": "audio", "language": "eng", "format": "audio/ogg"} ],
+  [ "2017-05-23T17:25:47-04:00", 20394823957, 923842093, "ae54db", "[35.678323, -108.25343]", {} ]
+]
+```
+
+# Specification
+
+The Flow Results specification builds on the [Data Package](https://specs.frictionlessdata.io/data-package/) specification. Thus a Flow Results Data Package must be a Data Package and conform to the Data Package specification.
+
+Additionally, a Flow Results data package must conform to these additional requirements:
+
+## Components
+
+A Flow Results data package consists of one [Descriptor](https://specs.frictionlessdata.io/data-package/#descriptor), which contains exactly one [Resource](https://specs.frictionlessdata.io/data-resource/) describing the interaction results.  
+
+## Descriptor
+
+The Descriptor JSON object must contain the following required metadata properties:
+
+<table>
+  <tr>
+    <td>Property</td>
+    <td>Description</td>
+    <td>Example</td>
+  </tr>
+  <tr>
+    <td>`profile`</td>
+    <td>Indicates this package is a Flow Results data package. Must be in the format of the example.</td>
+    <td>'flow-results-package'</td>
+  </tr>
+  <tr>
+    <td>`flow_results_specification_version`</td>
+    <td>Indicates the version of this specification the package is compliant with. The Flow Results specification adheres to semantic versioning.</td>
+    <td>'1.0.0-rc1'</td>
+  </tr>
+  <tr>
+    <td>`created`</td>
+    <td>The timestamp for when this package was created/published. This must be in the format of RFC 3339.</td>
+    <td>'2017-06-30 15:35:27+00:00'</td>
+  </tr>
+  <tr>
+    <td>`modified`</td>
+    <td>The latest timestamp for when the data in this package was refreshed/updated/changed. This must be in the format of RFC 3339.</td>
+    <td>'2017-06-30 15:38:05+00:00'</td>
+  </tr>
+</table>
+
+
+The following metadata properties are recommended, consistent with the Data Packages specification:  (TODO: Should any of these be required?)
```

Comment:
Good questions; let's chat about this tomorrow as it might help to get it figured out on the call. 

> More relevant to the issue of making these required, if they are not required, how does the user associate flow results data with a flow results descriptor?

 A descriptor and the data with it are inherently part of a package; I was assuming they get sent to the receiving system at the same time.

We could require the `id` field and stipulate that it is a globally unique id, like a UUID v4.